dxtbx/tests/tst_imageset.py

```diff
@@ -614,7 +614,39 @@ def run(self):
       g = iset.get_goniometer(i)
       s = iset.get_scan(i)
 
+class Test_SACLA_MPCCD_Cheetah_File(object):
```

Comment:
This test class should still be a function instead of a class. Test classes that only have ```__init__``` and a single function and no real state add unnecessary complexity